LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Remove this file

Reply:
@cpiggott MIT License still?

Reply:
What license do you want there @V-FEXrt ?

Reply:
I don't think we should have a License. Or at the very least we don't need a permissive one. Just because the repo is public, we don't need to give anyone access to use it

Reply:
Fair enough. Nuke it @ericschmar